plugins/CoreVisualizations/Visualizations/EvolutionPeriodSelector.php

```diff
@@ -30,15 +31,14 @@ public function getNumDaysDifference(Date $date1, Date $date2): int
         return (int) round($days);
     }
 
-    public function setSparklineDatePeriods($params, Period $originalPeriod, $comparisonPeriods): array
+    public function setDatePeriods($params, Period $originalPeriod, $comparisonPeriods, $isComparing): array
     {
-        $isComparingDates = !empty($comparisonPeriods);
         $highestPeriodInCommon = $this->getHighestPeriodInCommon($originalPeriod, $comparisonPeriods);
 
-        if ($isComparingDates) {
+        if ($isComparing) {
```

Comment:
When comparing periods, for consistency both sparklines and evolution graphs will now behave the same and show the evolution over the selected range instead of sparklines showing historical evolution.